docs/GUIDE.md

```diff
@@ -0,0 +1,123 @@
+# Step-by-step guide for Akka Serialization Helper usage
+This document is a guide on how to use Akka Serialization Helper in your project. If you want to get a more general view of this toolbox, see [README](../README.md).
+
+Akka Serialization Helper (ASH) consists of two major parts:
+1. Circe Akka Serializer
+2. Compiler Plugins
+
+You might use ASH in a few ways:
+- Use both parts of the toolbox (best option - gives you all benefits from the toolbox and maximum runtime safety)
+- Use Circe Akka Serializer but without enabling Compiler Plugins
+- Use Compiler Plugins but not Circe Akka Serializer (if you need to stick to your current Serializer)
+
+To use ASH (hole or chosen part), you need to first add Akka Serialization Helper in the `project/plugins.sbt` file and enable it in `build.sbt` - this is the standard way of using the toolbox. Just follow short instructions from [install part of README](https://github.com/VirtusLab/akka-serialization-helper#install).<br><br>
```

Comment:
_chosen_ doesn't sound well in such technical contexts, I see that the industry prefers _selected_

```suggestion
To use ASH (whole or a selected part), you need to first add Akka Serialization Helper in the `project/plugins.sbt` file and enable it in `build.sbt` - this is the standard way of using the toolbox. Just follow short instructions from [install part of README](https://github.com/VirtusLab/akka-serialization-helper#install).<br><br>
```